admin/billing/orb.go

```diff
@@ -30,6 +30,13 @@ const (
 
 var ErrCustomerIDRequired = errors.New("customer id is required")
 
+var planCache = make(map[string]planCacheEntry)
+
+type planCacheEntry struct {
+	planDisplayName string
+	lastUpdated     time.Time
+}
```

Comment:
Since this is updated with webhooks, it should be consistent, right? So can it be stored in Postgres instead? 

It would be useful to use Postgres because a) we have multiple replicas of the `admin` service, b) it makes introspection easier and provides easy access to the plan name in downstream pipelines